app/src/main/java/com/vero/tasky/core/presentation/components/RootNavigation.kt

```diff
@@ -2,25 +2,29 @@ package com.vero.tasky.core.presentation.components
 
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
 import androidx.navigation.NavHostController
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
+import com.vero.tasky.auth.presentation.login.LoginScreen
 import com.vero.tasky.core.presentation.navigation.Screens
 
 @Composable
 fun RootNavigation(
```

Comment:
Even tho this is in fact a Composable, so it's okey to have in in a "components" package

I personally like to have a package in the app, called "Navigation". And inside it I add this NavigationHost and the Screens object

It's another approach, not saying it's the correct one. Just giving you the option, so you use the one you like the most